apps/io_tester/io_tester.cc

```diff
@@ -772,7 +772,7 @@ class unlink_class_data : public class_data {
 
         return max_concurrent_for_each(boost::irange(UINT64_C(0), files_count()), max_concurrency(), [this] (uint64_t file_id) {
             const auto fname = get_filename(file_id);
-            const auto fsize = _config.file_size / files_count();
+            const auto fsize = align_up<uint64_t>(_config.file_size / files_count(), 4096u);
```

Comment:
Why not extent_size_hint_alignment as in previous patch?

Reply:
Fixed